### PR TITLE
[Personal contrib] JAMES-2056 Add default values for host & port in the CLI

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

**Comment by mbaechler** [comment]
So I was write : the code to handle default value was here. Just not working (:

File: server/container/cli/src/test/java/org/apache/james/cli/ServerCmdTest.java [hunk 1]
```diff
@@ -1267,6 +1267,41 @@ public void parseCommandLineShouldReturnACommandLineWithCorrectHostOption() thro
...
+    @Test
+    public void getPortShouldUseDefaultValueWhenNone() throws Exception {
+        String[] arguments = { "-h", "127.0.0.1", "command", "arg1", "arg2", "arg3" };
+        CommandLine commandLine = ServerCmd.parseCommandLine(arguments);
+        assertThat(ServerCmd.getPort(commandLine)).isEqualTo(9999);
+    }
+
+    @Test
+    public void getPortShouldUseDefaultValueWhenEmpty() throws Exception {
+        String[] arguments = { "-h", "127.0.0.1", "-p", "", "command", "arg1", "arg2", "arg3" };
```
Comment (chibenwa) [inline server/container/cli/src/test/java/org/apache/james/cli/ServerCmdTest.java]: What happens if I ommit **p** or **h**?

File: server/container/cli/src/main/java/org/apache/james/cli/ServerCmd.java [hunk 2]
```diff
@@ -68,16 +68,14 @@
...
     
     private static Options createOptions() {
-        Options options = new Options();
-        Option optHost = new Option(HOST_OPT_SHORT, HOST_OPT_LONG, true, "node hostname or ip address");
-        optHost.setRequired(true);
-        options.addOption(optHost);
-        options.addOption(PORT_OPT_SHORT, PORT_OPT_LONG, true, "remote jmx agent port number");
-        return options;
+        return new Options()
+                .addOption(HOST_OPT_SHORT, HOST_OPT_LONG, true, "node hostname or ip address")
```
Comment (chibenwa) [inline server/container/cli/src/main/java/org/apache/james/cli/ServerCmd.java]: What **true** means?

File: server/container/cli/src/test/java/org/apache/james/cli/ServerCmdTest.java (hunk 1 above)
Comment (chibenwa) [inline server/container/cli/src/test/java/org/apache/james/cli/ServerCmdTest.java]: Ok, already handled by the test suite.

**Comment by chibenwa** [comment]
Merged

**Pull request merged**